libc/test/include/CMakeLists.txt

```diff
@@ -33,3 +33,17 @@ if(LLVM_LIBC_FULL_BUILD AND libc.include.stdbit IN_LIST TARGET_PUBLIC_HEADERS)
       # to.
   )
 endif()
+
+# Similar situation applies to stdckdint_test.
+if(LLVM_LIBC_FULL_BUILD AND libc.include.stdckdint IN_LIST TARGET_PUBLIC_HEADERS)
```

Comment:
The test only uses the `include/llvm-libc-macros/stdckdint-macros.h` header, so you can just remove the dependency on `libc.include.stdckdint` and the check for `LLVM_LIBC_FULL_BUILD`.